### PR TITLE
docs: Updated inject example to show constructor usage

### LINKED ISSUE #163: Docs: Duplicate example for injecting service
## Describe the bug

On the [Usage](https://ngworker.github.io/lumberjack/docs/usage#using-the-lumberjackservice) page, the examples for injecting the service are both using the `inject` function instead of one of them using the constructor.

## To Reproduce

Visit: https://ngworker.github.io/lumberjack/docs/usage#using-the-lumberjackservice

## Expected behavior

The first example should mention a constructor-based example.

### PR DESCRIPTION
**Pull request by ajitzero**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Duplicate example in https://ngworker.github.io/lumberjack/docs/usage#using-the-lumberjackservice where both show usage with `inject()`.

Issue Number: N/A

## What is the new behavior?
Updated the first example to use the constructor for injection.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information
Fixes #163

### CODE REVIEW

File: packages/docs/lumberjack-docs-app/docs/usage.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ import { LumberjackService } from '@ngworker/lumberjack';
   // (...)
 })
 export class MyComponent implements OnInit {
-  readonly #lumberjack = inject(LumberjackService);
+  constructor(private readonly lumberjack: LumberjackService) {}
```
Comment (ajitzero) [inline packages/docs/lumberjack-docs-app/docs/usage.md:94]: Expand below and see [Line 110](https://github.com/ngworker/lumberjack/pull/162/files#diff-b9602f9aeee83dc8459120a15e94c5ac91537d447599084a42dd716628de145fR110) for comparison.

**Comment by NachoVazquez** [comment]
Thanks for the PR @ajitzero ! Very appreciated.

I'll take a look right now.

File: packages/docs/lumberjack-docs-app/docs/usage.md (hunk 1 above)
Comment (NachoVazquez) [reply packages/docs/lumberjack-docs-app/docs/usage.md:94]: praise: nice catch!

**Review by NachoVazquez — APPROVED** [review]
Thank you! ❤️

**Pull request merged**